src/components/carousel/carousel.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Image, Box } from 'grommet';
+
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import logos from '../../assets/logos/logos';
+
+const ResponsiveCarousel = () => {
+  const settings = {
```

Comment:
you can move those settings outside of the component since they do not depend on any props

Reply:
ok, move to another file or just outside of the component?

Reply:
just moving outside the component will be ok, this is needed because now you recreate those settings on every component rerender